docs/user-guide/containers.md

```diff
@@ -94,3 +94,7 @@ The relationship between Docker's capabilities and [Linux capabilities](http://m
 | SETFCAP |  CAP_SETFCAP |
 | WAKE_ALARM |  CAP_WAKE_ALARM |
 | BLOCK_SUSPEND |  CAP_BLOCK_SUSPEND |
+
+Here is an example pod that uses capabilities to allow a container to change its hostname:
```

Comment:
Since "Capabilities" is a Docker concept, we need to set it off from the plain English word; either cap the first letter, or specify "Docker capabilities" or something.